sway-fmt/src/traversal_helper.rs

```diff
@@ -79,6 +79,75 @@ pub fn format_data_types(text: &str) -> String {
     })
 }
 
+/// Formats Sway data types: Enums and Structs ...but _swayfully_
+pub fn format_swayful_data_types(text: &str) -> String {
```

Comment:
let's rename this to something like `format_align_data_types` (or if you have a better, descriptive name go for it) (naming is hard :) )

Reply:
Ok will do!